boefjes/boefjes/plugins/kat_ssl_certificates/main.py

```diff
@@ -9,9 +9,13 @@ def run(boefje_meta: BoefjeMeta) -> list[tuple[set, bytes | str]]:
     client = docker.from_env()
     input_ = boefje_meta.arguments["input"]
     hostname = input_["hostname"]["name"]
+    scheme = input_["ip_service"]["service"]["name"]
     ip_address = input_["ip_service"]["ip_port"]["address"]["address"]
     port = input_["ip_service"]["ip_port"]["port"]
 
+    if scheme != "https":
+        return [(set(), "")]
```

Comment:
I think we'd want to check for http (and possibly other services), and only then not scan as there are other services that we would like to check.
Returning an empty result is less usable than what we are currently doing for 'empty results':

`return [(set("info/boefje"), "Skipping check due to unsupported scheme.")]`

Reply:
Note that we should be careful with this `set` notation!
```shell
$ python -c 'print(set("info/boefje"))'
{'e', 'f', 'n', 'o', 'i', 'b', 'j', '/'}
```

Reply:
```suggestion
    if scheme == "http":
        return [({"info/boefje"}, "Skipping check due to non-TLS scheme")]
```

Reply:
Could you also flip the scheme check around?